src/components/CheckBox/index.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import styled from 'styled-components';
+import checkMark from '../../assets/properties/displayProperties/icons/checkmark.svg';
+
+function checkBox(props) {
+  return (
+    <StyledCheckBox {...props}>
+      <input
+        disabled={props.value === '(None)'}
+        type="checkbox"
+        id="checkbox"
+        className={props.value === '(None)' ? 'checkboxDisabled' : 'checkbox'}
```

Comment:
Why? You already have `[disabled]` to style upon

Reply:
True, edited.